### PR TITLE
Prev step button added for applying

### PR DESCRIPTION
**Pull request by ytdbuolf**
Prev-step button has been added to opening application flow
Issue #2445 has been fixed now

### CODE REVIEW

**Review by thesan — CHANGES_REQUESTED** [review]
Thank you for the PR. See the CR below

File: packages/ui/src/working-groups/modals/ApplyForRoleModal/ApplyForRoleModal.tsx [hunk 1]
```diff
@@ -291,6 +291,11 @@ export const ApplyForRoleModal = () => {
         </StepperModalWrapper>
       </StepperModalBody>
       <ModalFooter>
+        {/* <ButtonPrimary disabled={!form.formState.isValid} onClick={() => send('NEXT')} size="medium"> */}
+        <ButtonPrev onClick={() => send('PREV')} size="medium">
+          <Arrow direction="left" />
+          Prev step
+        </ButtonPrev>
```
[the remark this answers is not in the document]
Comment (freakstatic) [reply packages/ui/src/working-groups/modals/ApplyForRoleModal/ApplyForRoleModal.tsx:298]: @thesan it seems that @ytdbuolf was gone MIA go I apply your suggestion on this `[PR](https://github.com/Joystream/pioneer/pull/3249)`

**Comment by thesan** [comment]
Done by #3249

**Comment by thesan** [comment]
Done by #3249

**Pull request closed without merging**